### PR TITLE
Enable approvers to review report expenses from Slack itself

### PR DESCRIPTION
**Pull request by jatzz10**
### Context
As part of this 1st phase of this initiative, made few modifications in the report approval notifications message for approvers

Initiative - https://www.notion.so/fyleuniverse/Add-detailed-Report-View-for-report-approval-notifications-d5384286edf643d98b0cb408fc6962dd

ClickUp - https://app.clickup.com/t/1xbz27r

---
### Screenshots
Screenshot 2022-03-01 at 8 29 46 PM

---
**Note** - Modal related changes will be raised separately in another PR

### CODE REVIEW

File: fyle_slack_app/slack/interactives/block_action_handlers.py [hunk 1]
```diff
@@ -196,6 +198,10 @@ def handle_feedback_dialog(self, slack_payload: Dict, user_id: str, team_id: str
         return JsonResponse({})
 
 
+    def handle_report_expense_dialog(self, slack_payload: Dict, user_id: str, team_id: str) -> None:
+        pass
```
Comment (jatzz10) [inline fyle_slack_app/slack/interactives/block_action_handlers.py:202]: Rest of this modal logic will be handled in separate PR

File: fyle_slack_app/slack/interactives/block_action_handlers.py [hunk 2]
```diff
@@ -37,7 +38,8 @@ def _initialize_block_action_handlers(self):
             'report_paid_notification_preference': self.handle_notification_preference_selection,
             'report_commented_notification_preference': self.handle_notification_preference_selection,
             'expense_commented_notification_preference': self.handle_notification_preference_selection,
-            'open_feedback_dialog': self.handle_feedback_dialog
+            'open_feedback_dialog': self.handle_feedback_dialog,
+            'open_report_expense_dialog': self.handle_report_expense_dialog
```
Comment (shreyanshss7) [inline fyle_slack_app/slack/interactives/block_action_handlers.py:42]: Minor -> `open_report_expenses_dialog`

File: fyle_slack_app/slack/interactives/block_action_handlers.py (hunk 1 above)
Comment (shreyanshss7) [reply fyle_slack_app/slack/interactives/block_action_handlers.py:202]: Just return `JsonResponse({})`

File: fyle_slack_app/slack/ui/authorization/messages.py [hunk 3]
```diff
@@ -113,16 +113,26 @@ def get_post_authorization_message() -> List[Dict]:
...
                         "type": "plain_text",
-                        "text": "Approve",
+                        "text": ":rocket: Approve",
                     },
                     "value": "pre_auth_message_approve",
                     "action_id": "pre_auth_message_approve"
                 },
+                {
+                    "type": "button",
+                    "style": "primary",
```
Comment (shreyanshss7) [inline fyle_slack_app/slack/ui/authorization/messages.py:123]: If the style is primary the CTA would look green, is this in the design?

**Review by shreyanshss7 — APPROVED** [review]

File: fyle_slack_app/slack/ui/authorization/messages.py (hunk 3 above)
Comment (jatzz10) [reply fyle_slack_app/slack/ui/authorization/messages.py:123]: Yes, this is in the design

**Review by gokulk16 — APPROVED** [review]
